manager/integration/framework/steps/volume_steps.py

```diff
@@ -0,0 +1,75 @@
+from pathlib import Path
+import sys
+sys.path.append(str(Path(__file__).parents[1]))
+from volume.volume import Volume
+from utility import Utility
+from nodes import Nodes
+from node_exec import NodeExec
+
+class volume_steps:
```

Comment:
let's rename steps pkg folder to keywords to align with robot terminology. 

`keywords/volume.py` instead of `steps/volume_steps.py` (no need to have _steps, which is redundant.)

Reply:
refined.